rts/lra/lra-coordinator/src/main/java/io/narayana/lra/coordinator/domain/model/LRARecord.java

```diff
@@ -305,8 +309,10 @@ private int tryDoEnd(boolean compensate) {
                 // ask the participant to complete or compensate
                 Future<Response> asyncResponse = target.request()
                         .header(LRA_HTTP_CONTEXT_HEADER, lraId.toASCIIString())
+                        .header(LRA_HTTP_PARENT_CONTEXT_HEADER, parentId) // make the context available to participants
                         .header(LRA_HTTP_RECOVERY_HEADER, recoveryURI.toASCIIString())
-                        .property(LRA_HTTP_CONTEXT_HEADER, lraId) // make the context available to the jaxrs filters
+                        .property(LRA_HTTP_CONTEXT_HEADER, lraId) // make the context available to the jaxrs filters)
```

Comment:
It might be something I don't see in the diff but seems to have gained a spurious ")" at the end of the line